src/esp/assets/ResourceManager.h

```diff
@@ -431,12 +396,10 @@ class ResourceManager {
    * @brief Instantiate, or reinstatiate, PhysicsManager defined by passed
    * attributes
    * @param physicsManager The currently defined @ref physics::PhysicsManager.
-   * Will be reseted to the configured physics implementation.
-   * @param physicsFilename The physics configuration file from which to
-   * re-instatiate the @ref physics::PhysicsManager and parse object templates
-   * for the
-   * @ref physicsObjTemplateLibrary_. Defaults to the file location @ref
-   * ESP_DEFAULT_PHYS_SCENE_CONFIG set by cmake.*/
+   * Will be reseated to the configured physics implementation.
```

Comment:
"reseated"? typo?

Reply:
Actually it is intended to be "reseat".  Apparently it has a meaning with regard to smart pointers.   Surprised me too :).

https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#r35-take-a-shared_ptrwidget-parameter-to-express-that-a-function-might-reseat-the-shared-pointer